library/ssl_tls13_server.c

```diff
@@ -2617,7 +2617,8 @@ MBEDTLS_CHECK_RETURN_CRITICAL
 static int ssl_tls13_write_new_session_ticket_coordinate( mbedtls_ssl_context *ssl )
 {
     /* Check whether the use of session tickets is enabled */
-    if( ssl->conf->f_ticket_write == NULL )
+    if( ssl->conf->f_ticket_write == NULL ||
+        ssl->handshake->tls13_session_tickets == 0 )
```

Comment:
When `ssl->conf->f_ticket_write != NULL` and `ssl->handshake->tls13_session_tickets == 0` the debug message is wrong.